tests/view/test_view.py

```diff
@@ -87,3 +88,20 @@ def test_view_do_not_publish_invalid(self, runner):
             view.path.write_text("SELECT 1")
             assert view.is_valid() is False
             assert view.publish() is False
+
+    @patch("google.cloud.bigquery.Client", autospec=True)
+    def test_publish_valid_view(self, mock_bigquery_client):
+        mock_client = MagicMock()
+        mock_bigquery_client.return_value = mock_client
+        view = View.from_file(
+            TEST_DIR
+            / "data"
+            / "test_sql"
+            / "moz-fx-data-test-project"
+            / "test"
+            / "view_with_metadata"
+            / "view.sql"
+        )
+
+        assert view.is_valid()
+        assert view.publish()
```

Comment:
nit: feels like there should be an assert on the attributes of the view that gets passed to `get_table`/`update_table`.. Just to make it clear that the `friendly_name` and `description` have been attached